tests/debugging/function/test_discovery.py

```diff
@@ -12,7 +12,7 @@ def stuff_discovery():
 def test_abs_stuff():
     import tests.submod.absstuff as absstuff
 
-    assert sorted(FunctionDiscovery.from_module(absstuff).keys()) == [9, 13, 18, 21]
+    assert sorted(FunctionDiscovery.from_module(absstuff).keys()) == [7, 11, 16, 19]
```

Comment:
cc @P403n1x87 might be able to provide some insights, I assume these are line numbers, so if the source changes then the line numbers also need to be updated (similar for some profiling tests)

Reply:
Yep, the absstuff source was meant to have "append-only" changes. Since some lines at the top have been modified, we would expect this kind of issues to arise